src/Application/CommandHandler/CommentOrderHandler.php

```diff
@@ -20,21 +20,16 @@ final class CommentOrderHandler
     /** @var EntityManagerInterface */
     private $entityManager;
 
-    /** @var FilesystemInterface */
-    private $fileSystem;
-
     /** @var string */
     private $fileDir;
 
     public function __construct(
         OrderRepositoryInterface $orderRepository,
         EntityManagerInterface $entityManager,
-        FilesystemInterface $filesystem,
```

Comment:
What's the reason behind removing Gaufrette dependency? We lose an ability to store those files in a different filesystem this way.

Reply:
To be honest, I just didn't know about Gaufrette.